website/docs/reference/change-requests.mdx

```diff
@@ -10,171 +10,104 @@ import VideoContent from '@site/src/components/VideoContent.jsx';
 
 :::
 
+## Overview
 
-<VideoContent videoUrls={["https://www.youtube.com/embed/ENUqFVcdr-w"]}/>
-
-
-Feature flagging is a powerful tool, and because it's so powerful, you sometimes need to practice caution. The ability to have complete control over your production environment comes at the cost of the potential to make mistakes in production. Change requests were introduced in version 4.19.0 to alleviate this fear. Change requests allow you to group changes together and apply them to production at the same time, instead of applying changes directly to production. This allows you to make multiple changes to feature flags and their configuration and status (on/off) all at once, reducing the risk of errors in production.
+Change requests allow you to require an additional approval step before any changes can be made in an environment. This functionality supports the "four-eyes principle", ensuring compliance in industries with strict legal or regulatory requirements.
+ 
+Change requests also allow you to group changes and apply them at a [specific point in time](#scheduled-change-requests).
```

Comment:
should the link text be more of the sentence here? Feels like "apply them at a specifc point in time" is more descriptive of scheduled changes than "specific point in time". 
```suggestion
Change requests also allow you to group changes and [apply them at a specific point in time](#scheduled-change-requests).
```